### PR TITLE
Update vendor and version of the player schema used in the media plugin (close #1222)

### LINKED ISSUE #1222: Update vendor and version of the player schema used in the media plugin
The 1-0-0 version of the player schema has been removed from Iglu Central due to conflicts with another schema. [See this issue](https://github.com/snowplow/iglu-central/issues/1321).

The media plugin should use the following schema instead of the 1-0-0 version in all tracked media events:
`iglu:com.snowplowanalytics.snowplow/media_player/jsonschema/2-0-0`

### PR DESCRIPTION
**Pull request by matus-tomlein**
Issue #1222 

This PR updates the schema for the player entity used in the media plugin.

The previous schema had a conflict with another one which could lead to problems when loading data to warehouses. To face this, we republished the schema under the URI `iglu:com.snowplowanalytics.snowplow/media_player/jsonschema/2-0-0`.

### CODE REVIEW

**Review by igneel64 — COMMENTED** [review]
Goo 📺

**Review by igneel64 — APPROVED** [review]

File: plugins/browser-plugin-media/src/schemata.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { MediaEventType } from './types';
 
 const MEDIA_SCHEMA_PREFIX = 'iglu:com.snowplowanalytics.snowplow.media/';
 const MEDIA_SCHEMA_SUFFIX = '/jsonschema/1-0-0';
-export const MEDIA_PLAYER_SCHEMA = MEDIA_SCHEMA_PREFIX + 'player' + MEDIA_SCHEMA_SUFFIX;
+export const MEDIA_PLAYER_SCHEMA = 'iglu:com.snowplowanalytics.snowplow/media_player/jsonschema/2-0-0';
```
Comment (igneel64) [inline plugins/browser-plugin-media/src/schemata.ts:5]: We might want to leave a comment thought, WDYT ?

Comment (matus-tomlein) [reply plugins/browser-plugin-media/src/schemata.ts:6]: Good thinking, have added a comment!

**Review by greg-el — APPROVED** [review]
LGTM! 🚢

**Pull request merged**